publishing/entry-point-uri/README.md

```diff
@@ -0,0 +1,19 @@
+# Entry point URI
+
+Every blendable web app should have the entry point URI that follows the pattern `/<appname>`, e.g. `/kitchensink` (lowercase).
+
+This URI is automatically linked in Starcounter Administrator.
```

Comment:
I can promise that we've had! But now it seems it is not there. I will remove this line